plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/tasks/BaseKtLintCheckTask.kt

```diff
@@ -89,6 +90,10 @@ abstract class BaseKtLintCheckTask @Inject constructor(
 
     init {
         if (project.hasProperty(FILTER_INCLUDE_PROPERTY_NAME)) {
+            // if FILTER_INCLUDE_PROPERTY_NAME exists then we are invoked from a git hook, check hook version
+            if (project.findProperty(hookVersion) != hookVersion) {
+                throw GradleException("Your ktlint git hook is outdated, please update by running the addKtlint*GitPreCommitHook Gradle task.")
+            }
```

Comment:
I'd be inclined to make this a warning instead of an error. Or, if we do want to make this an error, you should be able to disable it to make it a warning instead.

@Tapchicoma thoughts?

Reply:
I've changed this to be a warning.

Reply:
If it's a warning then the user won't see it because `--quiet`.